src/test/resources/jsonld/HT018781534

```diff
@@ -30,6 +30,10 @@
     }
   },
   "edition" : [ "Score and parts" ],
+  "exampleOfWork" : {
+    "id" : "http://d-nb.info/gnd/107360974X",
+    "label" : "Momento capriccioso"
```

Comment:
It would be nice to also add the information from subfields `p` and `d` like we do e.g. at https://github.com/hbz/lobid-resources/pull/671/files#diff-5dd561891677f7164f7d7c47a1d0438b.